packages/components/src/Skeleton/__snapshots__/Skeleton.test.js.snap

```diff
@@ -46,6 +46,7 @@ exports[`Storyshots Skeleton default 1`] = `
       small
     </div>
     <span
+      aria-label=" circle (loading)"
```

Comment:
There special attribute to indicate that's loading, let's fix it in another PR!

Reply:
I changed it here, but not other component fixes, because there is an action that renders a skeleton if it's loading.
This PR is focused on Actions components, but the skeleton was part of one of them. This fixes this Action.

Reply:
We've to don't repeat ourself with this skeleton, good to know.